### PR TITLE
MF-312 - Implement basic MongoDB reader

### LINKED ISSUE #312: Implement basic MongoDB reader
A basic reader that only reads messages and has a pagination should be implemented to facilitate experimenting with and demoing Mainflux system.

Reader exposes only one HTTP REST API endpoint to read messages from DB.

Something similar existed before for MongoDB: https://github.com/MainfluxLabs/mainflux-mongodb-reader

### PR DESCRIPTION
**Pull request by anovakovic01**
This pull request resolves #312.

**Summary:**
- Add mongodb reader service
- Add docs for mongodb reader service
- Add tests for mongodb reader service

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: docker/addons/mongodb-reader/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+###
+# This docker-compose file contains optional MongoDB-reader service
+# for Mainflux platform. Since these are optional, this file is dependent of docker-compose file
+# from <project_root>/docker. In order to run these optional service, execute command:
+# docker-compose -f docker/docker-compose.yml -f docker/addons/mongodb-reader/docker-compose.yml up
+# from project root. MongoDB default port (27017) is exposed, so you can use various tools for database
```
Comment (dborovcanin) [inline docker/addons/mongodb-reader/docker-compose.yml]: This last sentence can be removed.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Comment by drasko** [comment]
Merged! Thanks @anovakovic01

**Pull request merged**